docs/platform/howto/manage-organization-vpc.md

```diff
@@ -0,0 +1,323 @@
+```bash
+avn service create SERVICE_NAME        \
+  --project PROJECT_NAME               \
+  --organization-vpc-id ORGANIZATION_VPC_ID \
+  --type SERVICE_TYPE                  \
+  --plan SERVICE_PLAN                  \
+  --cloud CLOUD_PROVIDER_REGION
+```
+
+Replace the following:
+
+- `SERVICE_NAME` with the name of the service to be created, for example,
+  `pg-vpc-test`
+- `PROJECT_NAME` with the name of the project where to create the service, for example,
+  `pj-test`
+- `ORGANIZATION_VPC_ID` with the ID of your organization VPC, for example,
+  `12345678-1a2b-3c4d-5f6g-1a2b3c4d5e6f`
+- `SERVICE_TYPE` with the type of the service to be created, for example, `pg`
+- `SERVICE_PLAN` with the plan of the service to be created, for example, `hobbyist`
+- `CLOUD_PROVIDER_REGION` with the cloud provider and region to host the service to be
+  created, for example `aws-eu-west-1`
+
+</TabItem>
+-->
+<TabItem value="api" label="Aiven API">
+
+Make an API call to the
+[ServiceCreate endpoint](https://api.aiven.io/doc/#tag/Service/operation/ServiceCreate)
+endpoint:
+
+```bash {12}
+curl --request POST \
+  --url https://api.aiven.io/v1/project/PROJECT_NAME/service \
+  --header 'Authorization: Bearer BEARER_TOKEN' \
+  --header 'content-type: application/json' \
+  --data-raw '
+    {
+      "service_name": "SERVICE_NAME",
+      "cloud": "CLOUD_PROVIDER_REGION",
+      "plan": "SERVICE_PLAN",
+      "service_type": "SERVICE_TYPE",
+      "disk_space_mb": DISK_SIZE,
+      "organization_vpc_id":"ORGANIZATION_VPC_ID"
+    }
+  '
+```
+
+Replace the following placeholders with meaningful data:
+
+- `PROJECT_NAME`, for example `org-vpc-test`
+- `BEARER_TOKEN`
+- `SERVICE_NAME`, for example `org-vpc-test-project`
+- `CLOUD_PROVIDER_REGION`, for example `google-europe-west10`
+- `SERVICE_PLAN`, for example `startup-4`
+- `SERVICE_TYPE`, for example `pg`
+- `DISK_SIZE` in MiB, for example `81920`
+- `ORGANIZATION_VPC_ID`
+
+</TabItem>
+</Tabs>
+
+## Migrate a service to an organization VPC
+
+Your organization VPC is available as a geolocation (cloud region) for your service.
+
+:::note
+You can only migrate a service to an organization VPC if the organization VPC is located in
+the organization where your service runs.
+:::
+
+Migrate a service to an organization VPC using a tool of your choice:
+
+<Tabs groupId="group1">
+<TabItem value="console" label="Aiven Console" default>
+
+1. In [Aiven Console](https://console.aiven.io/), open your service and click
+   <ConsoleLabel name="Service settings"/>.
+1. In the **Cloud and network** section, click <ConsoleLabel name="actions"/> >
+   **Change cloud or region**.
+1. In the **Region** section, go to the **VPCs** tab, select your organization VPC and
+   click **Migrate**.
+
+</TabItem>
+<!--
+<TabItem value="cli" label="Aiven CLI">
+
+Run [avn service update](/docs/tools/cli/service-cli#avn-cli-service-update):
+
+```bash
+avn service update SERVICE_NAME \
+  --organization-vpc-id ORGANIZATION_VPC_ID
```

Comment:
In the service API here we still have `project-vpc-id` here and `project_vpc_id` in here, even if the value can be an ID of an organization ID too. This is for API compatibility as a rename in API would be a breaking change, so I elected to not do it.